continuous_integration/travis/install.sh

```diff
@@ -69,19 +69,23 @@ create_new_conda_env() {
 
 create_new_conda_env
 
+# Install py.test timeout to fasten failure in deadlocking tests
+PIP_INSTALL_PACKAGES="pytest-timeout"
```

Comment:
a bit unrelated

Reply:
I just refactored all the pip install in one so I had to put this one in the list.
I think it makes think easier to read but I can put it directly in the `pip install` if you prefer.